source/renderer/app/components/voting/voting-info/VotingInfo.scss

```diff
@@ -4,9 +4,11 @@
   flex: 1 0 0;
   padding: 20px;
 
-  .separator {
+  hr {
```

Comment:
Wouldn't be better to use class name instead of tight coupling the component markup with styling?

Reply:
Right, fixed. Thanks!